sktime/forecasting/model_evaluation/_functions.py

```diff
@@ -44,7 +44,8 @@ def evaluate(
         Exogenous variables
     strategy : {"refit", "update"}
         Must be "refit" or "update". The strategy defines whether the `forecaster` is
-        only fitted on the first train window data and then updated, or always refitted.
+        only fitted on the first train window data and then updated,
```

Comment:
why did you shorten this?

Reply:
THe rest of the sentence is on a new line. I must've moved it becuase of fear the sentence would exceed max line number. I have moved it back.

Reply:
Thanks! It does not look like you have moved it back though?

Max line length is 88 characters for `sktime`, see `setup.cfg`.
If you are using vscode and have linting turned on, it should automatically watch out for that.

You can also add rulers to your editor windows:
https://stackoverflow.com/questions/29968499/vertical-rulers-in-visual-studio-code

Reply:
On line 47 of commit [38c3351](https://github.com/alan-turing-institute/sktime/pull/2850/commits/38c3351f39ed14b254fb1a624e8d79993b521d51), i think it's back. This commit is included in [1d6361b](https://github.com/alan-turing-institute/sktime/pull/2850/commits/1d6361b727f9ec12a36e66a6bf73adab19dc60eb), which is being tested. Do you see the full sentence there?